pkgs/by-name/tl/tldx/package.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, nix-update-script }:
+
+buildGoModule (finalAttrs: rec {
+  pname = "tldx";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "brandonyoungdev";
+    repo = pname;
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-MEFj8GaHXoDo2rAP+tOnisgLSZDqleh4pzX7M4CZBUk=";
+  };
+
+  vendorHash = "sha256-+I1fFsHnOMK8oW810fj2cmD3jdunSFL/btaxLZCXcts=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=v${finalAttrs.version}"
+    "-X main.commit=${finalAttrs.version}"
+    "-X main.date=1970-01-01"
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
```

`with lib;` obscures where variables come from and introduce unintuitive scoping rules. It's best to avoid it and prefix variables explicitly 

```suggestion
  meta = {
```